trace/trace_otel.go

```diff
@@ -0,0 +1,26 @@
+package trace
```

Comment:
can this go in `internal/trace`

Reply:
yup - I'm re-working it, it will be embedded in server startup